lib/rex/post/meterpreter/ui/console/command_dispatcher/stdapi/ui.rb

```diff
@@ -168,6 +170,166 @@ def cmd_screenshot( *args )
   end
 
   #
+  # Screenshare the current interactive desktop.
+  #
+  def cmd_screenshare( *args )
+    stream_path    = Rex::Text.rand_text_alpha(8) + ".jpeg"
+    player_path = Rex::Text.rand_text_alpha(8) + ".html"
+    quality = 50
+    view    = false
+    duration = 1800
+
+    screenshare_opts = Rex::Parser::Arguments.new(
+      "-h" => [ false, "Help Banner." ],
+      "-q" => [ true, "The JPEG image quality (Default: '#{quality}')" ],
+      # "-p" => [ true, "The JPEG image path (Default: '#{path}')" ],
+      "-v" => [ true, "Automatically view the JPEG image (Default: '#{view}')" ],
+      "-d" => [ true, "The stream duration in seconds (Default: 1800)" ] # 30 min
+    )
+
+    screenshare_opts.parse( args ) { | opt, idx, val |
+      case opt
+        when "-h"
+          print_line( "Usage: screenshare [options]\n" )
+          print_line( "View the current interactive desktop in real time." )
+          print_line( screenshare_opts.usage )
+          return
+        when "-q"
+          quality = val.to_i
+        when "-p"
+          path = val
+        when "-v"
+          view = true if ( val =~ /^(t|y|1)/i )
```

Comment:
@timwr - line 202 here should changed to "view = false if ( val =~ /^(f|n|0|no|false)/i )".